src/ctelnet.cpp

```diff
@@ -1914,6 +1914,11 @@ void cTelnet::handle_socket_signal_readyRead()
         }
         else
         {
+            if( ch == TN_BELL )
```

Comment:
Well, that hard codes a specific (flash for three seconds) behaviour that the user cannot turn off or modify!  Also there are some ANSI ESC sequences that use that character as the end marker if I recall correctly - not sure that Mudlet supports them but I thought it was something to do with a "set terminal/window name" or possibly an OSC (as opposed to SCR) code for the remote connection (e.g. the Server in our case) to define colour settings... I do not know whether such codes would get this far in the processing {but it _is_ upstream of the `TBuffer::translateToPlainText(...)` that I've been hacking recently so that is possible I suppose...}

Other than this I am happy with the PR... 😃